go/cmd/sqlflowserver/e2e_mysql_test.go

```diff
@@ -215,6 +216,16 @@ func TestEnd2EndMySQL(t *testing.T) {
 	t.Run("CaseTestOptimizeClauseWithoutConstraint", caseTestOptimizeClauseWithoutConstraint)
 }
 
+func caseScoreCard(t *testing.T) {
+	a := assert.New(t)
+	sql := `SELECT * FROM scorecard.train LIMIT 2000
```

Comment:
Should we popularize the SQL statement to generate the table `scorecard.train`?

Reply:
I think so, the tutorial in the future would use this dataset.